test/Test/Data/List.purs

```diff
@@ -398,6 +407,27 @@ testList = do
   log "append should be stack-safe"
   void $ pure $ xs <> xs
 
+  log "mapReverse should work the same as slower naive version"
+  --quickCheck mapReverseLaw
+
+  log "naiveAddIndexReverse should work the same as slower naive version"
+  --quickCheck naiveAddIndexReverse
+
+  log "nubByAdjacentReverse should be correct"
+  assert $ nubByAdjacentReverse (on eq Array.length) ([1]:[2]:[3,4]:Nil) == [3,4]:[2]:Nil
+
+naiveMapReverse :: forall a b. (a -> b) -> List a -> List b
+naiveMapReverse f l = map f l # reverse
```

Comment:
For consistency, could you please use a right-to-left style here (and elsewhere)? So prefer `<<<` over `>>>`, and `$` over `#`.

Reply:
Will do.
Note that the new `nubBy` uses lots of `>>>`, which I think is easier to read. Should that be changed to `<<<`?

Reply:
For that one I could go either way, so I wouldn't mind leaving it as-is. Wherever it's a simple enough case that either is equally clear, though, I think we should use right-to-left style.

Reply:
Yes please